source/common/filter/reader/example/BUILD

```diff
@@ -0,0 +1,43 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
FYI, we're considering reworking how filters are organized in the main Envoy repo. See https://github.com/envoyproxy/envoy/issues/2069. Do we want to get some informal buy-in to the new layout in that issue now and have this reflect that?

Reply:
Seems reasonable, let me try to reflect that in this example.

Reply:
You might want to comment on the issue that you're doing this to allow those who have reservations to speak now or forever hold their peace :)